sencha-workspace/packages/slate-cbl/src/view/teacher/DashboardController.js

```diff
@@ -222,11 +242,145 @@ Ext.define('Slate.cbl.view.teacher.DashboardController', {
             });
         });
     },
+    
+    onDemonstrationDelete: function(demonstration, operation) {
+        var me = this,
+            dashboardView = me.view,
+            mainGridEl = dashboardView.el.down('.cbl-grid-main'),
```

Comment:
that would be good but we'd have to hook up a render listener on '#' to initialize it, I don't think its too performance sensitive here though